setup.py

```diff
@@ -37,10 +40,23 @@ def run(self):
         develop.run(self)
 
 
+def media_server_test_suite():
```

Comment:
Can you explain what is going on here? why is `./setup.py test` or `nosetests` enough?